### ISSUE TITLE
Utility pole with electrical breaker

### ISSUE THREAD

**Issue by bowserjr2424** [Electrical-Age/ElectricalAge#591]
After frying the grid on erisia 14, I have a good solution. There should be poles that have breakers on them, so very bad things don't happen

**Comment by AMIDIBOSS** [comment]
Alright, good idea. And what did you do that you fried the whole grid?

**Comment by bowserjr2424** [comment]
I was messing w/ a transformer

**Comment by cm0x4D** [comment]
Imho the best would be to add a high voltage switchgear similar to [this one](https://www.youtube.com/watch?v=C-UTYzfNjKM), where it would be possible to install different controllers in order to control open and close the switchgear. One could be a purely manual control, then an over-current protection, an over-voltage protection and maybe an advanced device that can be configured.

**Comment by cm0x4D** [comment]
The protection devices could look somehow like [this](http://new.abb.com/substation-automation/products/protection-control/line-distance-protection/rel670) one from ABB...

**Comment by AMIDIBOSS** [comment]
Yeah, if you could add those effects :)  Wait, its not that hard. I think i can do this! 
And that electrical box on the pole pretending to be dc converter... Maybe we can add gui to it? You could still add transformer, but also breakers n stuff.

**Comment by bowserjr2424** [comment]
@AMIDIBOSS do you play on erisia, if so, what is your username
